docs/index.bs

```diff
@@ -3014,7 +3033,9 @@ spec: rfc7231; urlPrefix: https://tools.ietf.org/html/rfc7231
           1. Else:
               1. If |request|'s [=request/url=] is not a <a>potentially trustworthy URL</a>, return null.
           1. If |request| is a <a>navigation request</a> and the <a lt="navigate">navigation</a> triggering it was initiated with a shift+reload or equivalent, return null.
-          1. Set |registration| to the result of running <a>Match Service Worker Registration</a> algorithm passing |request|'s [=request/url=] as the argument.
+          1. [=Assert=] |reservedClient| is not null.
```

Comment:
There is a check for null `reservedClient` above, but I don't understand when that can happen.  And we don't have a way to get the storage key without a client.  Any thoughts on best thing to do here?